sdk/src/Core/ThirdParty/LitJson/IJsonWrapper.cs

```diff
@@ -1,3 +1,5 @@
+#pragma warning disable CS1587 // XML comment is not placed on a valid language element
```

Comment:
Why does this warning needs to be suppressed? I don't see any XML comment on the namespace (or anywhere else in this file) that would raise this warning.

Wondering the same for other places in the `LitJson` files.

Reply:
It's the lack of the Amazon Copyright notice.  I'm assuming in this case Amazon can't claim notice for these files since they were pulled from somewhere else.

Reply:
Right. This code was ported directly from LitJson so we can't put Amazon's Copyright notice.